### ISSUE TITLE
Accessibility - My Profile Screen - Input Field is Not Telling VoiceOver User It's Use

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#12]
On the preliminary "My Profile" screen VoiceOver is reading the “Office Address” section as four separate INPUT fields? The fields either should be one large field to enter all the address information or at least define what part of the address they are in such as address line 1, address line 2, city, state and zip code.

**Comment by aanthonyDOL** [comment]
On the My Profile Screen I've added Voice Over to define what content the field contains

**Comment by jfc3-dol** [comment]
@aanthonyDOL,

When I remove the office address information from the field to see what happens a comma shows up on the screen with no other content in the address section of the screen. VoiceOver reads "Office Address dimmed" for the first line then "Comma dimmed" for the second line.

Please see the attached screenshot.

![safetynet - profile screen - after removing address content it shows a comma and reads dimmed comma dimmed](https://user-images.githubusercontent.com/10423487/62657150-d2a4d800-b933-11e9-852b-a1349e83a1a9.png)
